.travis.yml

```diff
@@ -1,67 +1,34 @@
-# We set the language to c because python isn't supported on the MacOS X nodes
-# on Travis. However, the language ends up being irrelevant anyway, since we
-# install Python ourselves using conda.
+# Most of the CI testing happens on Azure, but this configuration is for a
+# regular cron job on Travis (as well as for pull requests).
+
 language: c
 
 os:
     - osx
     - linux
 
-# Adding notifications since this is a project we otherwise don't update
-# often, but need to make sure we see failures when they happen when
-# running cron jobs.
-notifications:
-  email:
-    recipients:
-      - thomas.robitaille@gmail.com
-
 # Setting sudo to false opts in to Travis-CI container-based builds.
 sudo: false
 
-# The apt packages below are needed for sphinx builds, which can no longer
-# be installed with sudo apt-get.
 addons:
     apt:
         packages:
             - graphviz
 
 env:
-  matrix:
-    - PYTHON_VERSION=2.7
-    - PYTHON_VERSION=3.4 SPHINX_VERSION=1.3
-    - PYTHON_VERSION=3.5 SPHINX_VERSION=1.4
-    - PYTHON_VERSION=3.5 SPHINX_VERSION=1.5
-    - PYTHON_VERSION=3.6 SPHINX_VERSION=1.6
-    - PYTHON_VERSION=3.6 SPHINX_VERSION=1.7
-    - PYTHON_VERSION=3.7 SPHINX_VERSION=1.8
-    - PYTHON_VERSION=3.7 SPHINX_VERSION=2.0
-    - PYTHON_VERSION=3.7 SPHINX_VERSION=dev CONDA_DEPENDENCIES="setuptools cython pytest-cov"
-    - PYTHON_VERSION=2.7 LOCALE=C
-    - PYTHON_VERSION=3.7 LOCALE=C
   global:
-    - LOCALE=default
-    - CONDA_DEPENDENCIES="setuptools sphinx cython pytest-cov"
+    - CONDA_DEPENDENCIES="setuptools cython pytest-cov"
     - PIP_DEPENDENCIES="codecov"
     - HOMEBREW_NO_AUTO_UPDATE=1
+    - PYTHON_VERSION=3.7
 
 before_install:
-
-    # Make sure that things work even if the locale is set to C (which effectively means ASCII).
-    # Some of the input rst files have unicode characters and we need to deal with this gracefully.
-    - if [[ $LOCALE == C ]]; then
-        export LC_CTYPE=C;
-        export LC_ALL=C;
-        export LANG=C;
-      fi
-
     - if [[ $TRAVIS_OS_NAME == osx ]]; then brew install graphviz; fi
 
 install:
     - git clone git://github.com/astropy/ci-helpers.git
     - source ci-helpers/travis/setup_conda.sh
-    - if [[ $SPHINX_VERSION == dev ]]; then
-        pip install git+https://github.com/sphinx-doc/sphinx.git;
-      fi
+    - pip install git+https://github.com/sphinx-doc/sphinx.git
 
 script:
     - py.test --cov sphinx_automodapi sphinx_automodapi -v
```

Comment:
no need to run coverage here either, not sure it would nicely interact between the two CI systems.

Reply:
codecov does merge them automatically and it seems to work :)

Reply:
I'll remove though since it's now cron-only and Azure won't necessarily run at the same time